packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -813,7 +813,8 @@ export const Table = <
                             )}
                             {showSortButton(column) && (
                               <div className='iui-cell-end-icon'>
-                                {column.isSorted && column.isSortedDesc ? (
+                                {(column.isSorted && column.isSortedDesc) ||
+                                (!column.isSorted && column.sortDescFirst) ? (
```

Comment:
Do you think we need `isSorted` here from your testing?
Wouldnt it be enough to have `column.isSortedDesc || column.sortDescFirst` ?

Reply:
It is necessary. You can't say `isSorted && (isSortedDesc || sortDescFirst)` because if you're sorting desc first, the column won't be sorted initially when the down icon should appear, and therefore will fail the `isSorted` check. And you can't say `(isSorted && isSortedDesc) || sortDescFirst` because then the down icon will always appear for columns sorted desc first.

Reply:
sorry, you might have misunderstood, but I am asking if you were to remove `isSorted` from both conditions.
So instead of `(column.isSorted && column.isSortedDesc) || (!column.isSorted && column.sortDescFirst)` have just `column.isSortedDesc || column.sortDescFirst`.

I understand why your first given example woudl not work, but I am not sure what you had in mind with the 2nd one.

Reply:
Seems like `column.isSortedDesc || (!column.isSorted && column.sortDescFirst)` works though